lib/WWW/Spotify.pm

```diff
@@ -159,6 +159,34 @@ has ua => (
     },
 );
 
+has 'response_status' => (
+    is      => 'rw',
+    isa     => Str,
+    default => q{}
+);
+
+has 'response_content_type' => (
+    is      => 'rw',
+    isa     => Str,
+    default => q{}
+);
+
+has 'request_custom_handler' => (
+    is      => 'rw',
+    default => ''
+);
+
+has 'request_custom_handler_result' => (
+    is      => 'rw',
+    default => ''
+);
+
+has 'check_response' => (
+    is      => 'rw',
```

Comment:
Wondering why this is `rw`? Do you want to allow the user to toggle this on/off after object creation?

Reply:
allowing it to be dynamic made the most sense to me, any specific concerns?

Reply:
No, just wanted to make sure that this was your intent.